packages/zipkin-instrumentation-redis/src/zipkinClient.js

```diff
@@ -42,24 +43,44 @@ module.exports = function zipkinClient(
     'slaveof',
     'config',
     'sentinel'];
-  methodsToWrap.forEach((method) => {
-    if (restrictedCommands.indexOf(method) > -1) {
-      return;
-    }
-    const actualFn = redisClient[method];
-    redisClient[method] = function(...args) {
-      const callback = args.pop();
-      let id;
-      tracer.scoped(() => {
-        id = tracer.createChildId();
-        tracer.setId(id);
-        commonAnnotations(method);
-      });
-      const wrapper = mkZipkinCallback(callback, id);
-      const newArgs = [...args, wrapper];
-      actualFn.apply(this, newArgs);
-    };
-  });
+  const wrap = function(client, parentId) {
```

Comment:
Is `parentId` actually the parent in the trace or is it the shared trace id across multi/batch calls?

Reply:
you are right, this is traceId, I fixed it in the latest commit